modules/nextrollBidAdapter.js

```diff
@@ -0,0 +1,234 @@
+import * as utils from '../src/utils';
```

Comment:
Can you update to add 'js' extension to all imports. We recently did a change here https://github.com/prebid/Prebid.js/pull/4876